.github/workflows/slack-notify.yml

```diff
@@ -0,0 +1,18 @@
+name: Notifications
+
+on:
+  workflow_call:
+
+
+jobs:
+  slack:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post to a Slack channel
+        id: slack
+        uses: slackapi/slack-github-action@v1.25.0
+        with:
+            channel-id: 'C06B3H23Z1N'
```

Comment:
Let's make this a variable. I'll make one called `SLACK_CHANNEL_ID`

Reply:
updated